daemon/container_unix.go

```diff
@@ -1007,13 +1013,15 @@ func (container *Container) connectToNetwork(idOrName string, updateSettings boo
 		}
 	}()
 
-	if err := container.updateEndpointNetworkSettings(n, ep); err != nil {
+	if err = container.updateEndpointNetworkSettings(n, ep); err != nil {
```

Comment:
Uh-oh, that's why I'm fan of named return handling in defers. Because someone new will write `:=` 100%